src/styles/_header.scss

```diff
@@ -37,5 +39,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $transition-duration;
```

Comment:
u could also add variables for transition property and colors, but i dont think its necessary in this task)